public/views/index.html

```diff
@@ -121,6 +121,16 @@ <h3 id="slogan">What is Cards for Humanity?</h3>
               </span>
         </span>
        </div>
+       <div class="text-center facebook-like-container ">
+        <div class="col-md-4">
+        </div>
+       <div class="col-md-4 text-center facebook-like">
+       <a href="https://www.facebook.com/Cards-For-Humanity-701709783330407/"> Visit Our Facebook Page</a>
+             <iframe src="https://www.facebook.com/plugins/like.php?href=https%3A%2F%2Fwww.facebook.com%2FCards-For-Humanity-701709783330407%2F&width=450&layout=standard&action=like&show_faces=true&share=false&height=80&appId" width="450" height="80" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true"></iframe>
+</div>
+ <div class="col-md-4">
```

Comment:
Thanks, it's actually important to complete the 12column grid implementation in bootstrap.